deploy/crds/operator.tigera.io_v1_amazoncloudintegration_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operator.tigera.io/v1
+kind: AmazonCloudIntegration
+metadata:
+  name: example-amazoncloudintegration
+spec:
+  # Add fields here
```

Comment:
And remove this and the following line.
Since an empty ACI CR isn't valid I guess you should add the fields here and add dummy values.

Reply:
done